### ISSUE TITLE
is operator doesn't work with qualified paths of secondary module types

### ISSUE THREAD

**Issue by Gama11** [HaxeFoundation/haxe#10174]
```haxe
function main() {
	// Module Foo does not define type Foo
	"" is Foo.Bar;

	// compiles fine
	Std.isOfType("", Foo.Bar);
}
```

`Foo.hx`:

```haxe
class Bar {}
```

____

`is Foo.Bar` also compiles fine if `Foo.hx` also contains a `class Foo`, or with `import Foo.Bar` and `is Bar`...

**Comment by nadako** [comment]
I can't reproduce this.

**Comment by Gama11** [comment]
Huh, interesting... I pushed the repro to this branch: https://github.com/Gama11/HaxeRepro/tree/haxe-10174

**Comment by RealyUniqueName** [comment]
Fixed by 996c39daf0fa4bebf8a4fce1f57ad8087ef69a3e
